src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java

```diff
@@ -2033,9 +2033,11 @@ private Message handleFetchFailure(VAdminProto.HandleFetchFailureRequest handleF
         } else {
             FailedFetchLock distributedLock = null;
             try {
+                logger.info("Try to acqure a distributed lock.");
                 distributedLock = FailedFetchLock.getLock(voldemortConfig, new Props(extraInfoProperties));
 
                 distributedLock.acquireLock();
+                logger.info("Distributed lock acquired.");
```

Comment:
I believe the lock acquired message belongs in the *Lock class, along with the time taken.